Interop/Indexer/prebuilt/nativeInteropStubs/cpp/signalChaining.cpp

```diff
@@ -0,0 +1,58 @@
+#if defined(__linux__) || defined(__APPLE__)
+#include <dlfcn.h>
+#include <stdio.h>
+#include <signal.h>
+#if defined(__linux__)
+#include <link.h>
+#endif
+
+typedef void (*toggleCrashRecovery_t)(int);
+typedef int (*sigaction_t)(int, const struct sigaction *, struct sigaction *);
+
+static sigaction_t realSigaction = 0;
+
+static int mySigaction(int sig, const struct sigaction *act, struct sigaction * oact) {
```

Comment:
Why not use `libjsig`?

Reply:
It makes no sense. If libjsig is not preloaded, JVM doesn't know about it. Also this way it's much simpler and has no path assumptions.